R/readWriteMgfData.R

```diff
@@ -83,6 +102,11 @@ writeMgfContent <- function(sp, TITLE = NULL, con) {
         }
     } else .cat("\nRTINSECONDS=", rtime(sp))
 
+    if (length(addFields)) {
+        if (!is.null(names(addFields)))
+            .cat(paste0("\n", paste(toupper(names(addFields)),
+                                    addFields, sep = "="), collapse = ""))
```

Comment:
`.cat("\n", paste(toupper(names(addFields)), addFields, sep = "=", collapse = "\n"))`

Reply:
yes, better that way. Thanks.